scripts/build-index.js

```diff
@@ -44,7 +55,8 @@ function buildPagesIndex(files) {
         return {
           name: page,
           platform: obj[page]["platform"],
-          language: obj[page]["language"]
+          language: obj[page]["language"],
```

Comment:
Also here I see that `["key"]` is used while in the rest of the script the dot is used instead (`.key`). Should these lines be refactored to use the dot?

Reply:
I don't mind.

Reply:
Personally I'd refactor, but it isn't a big deal.